src/go/rpk/pkg/adminapi/admin.go

```diff
@@ -114,3 +126,45 @@ func NewHostClient(fs afero.Fs, p *config.RpkProfile, host string) (*rpadmin.Adm
 	}
 	return rpadmin.NewClient(addrs, tc, auth, p.FromCloud)
 }
+
+// licenseFeatureChecks checks if the user is talking to a cluster that has
+// enterprise features enabled without a license and returns a message with a
+// warning.
+func licenseFeatureChecks(ctx context.Context, fs afero.Fs, cl *rpadmin.AdminAPI, p *config.RpkProfile) string {
+	var msg string
+	// We only do a check if:
+	//   1. LicenseCheck == nil: never checked before OR last check was in
+	//      violation. (we only save successful responses).
+	//   2. LicenseStatus was last checked more than 7 days ago.
+	if p.LicenseCheck == nil || p.LicenseCheck != nil && time.Unix(p.LicenseCheck.LastUpdate, 0).AddDate(0, 0, 7).Before(time.Now()) {
+		resp, err := cl.GetEnterpriseFeatures(ctx)
+		if err != nil {
+			zap.L().Sugar().Warnf("unable to check licensed enterprise features in the cluster: %v", err)
+			return ""
+		}
+		// We don't write a profile if the config doesn't exist.
+		y, exists := p.ActualConfig()
+		var licenseCheck *config.LicenseStatusCache
+		if resp.Violation {
+			var features []string
+			for _, f := range resp.Features {
+				if f.Enabled {
+					features = append(features, f.Name)
+				}
+			}
+			msg = fmt.Sprintf("You're using Enterprise features in your Redpanda cluster: %v. These features require a license, visit https://docs.redpanda.com/current/get-started/licenses for more information.", features)
```

Comment:
```suggestion
			msg = fmt.Sprintf("A Redpanda Enterprise Edition license is required to use enterprise features: %v. See https://docs.redpanda.com/current/get-started/licenses for more information.", features)
```

Reply:
LGTM

Reply:
Fixed, thanks!

Reply:
Sorry, reading this again..  I like the original text which is more clear that the system you're using is actually using these features..

**'WARNING: The following Enterprise features are being used in your Redpanda cluster: [feature1, feature 2]. These features require a license.  To get a license, Contact us at https://www.redpanda.com/contact.   For more information, see ttps://docs.redpanda.com/current/get-started/licenses"**